### ISSUE TITLE
"type_" property ignored

### ISSUE THREAD

**Issue by couchemar** [ahrefs/bs-reactstrap#8]
This code: 
```
<Input type_="checkbox" />
```

is transformed to this HTML:
```
<input type_="checkbox" type="text">
```

It looks like we need add `[@bs.as "type"]` to `type_`, but i don't know how to do it with current `makeProps` function: https://github.com/ahrefs/bs-reactstrap/blob/9cfd352716bf652e7618e5739b917692531e855f/src/BsReactstrap__Input.re#L5
NB
Probably the same problem: https://github.com/ahrefs/bs-reactstrap/blob/9cfd352716bf652e7618e5739b917692531e855f/src/BsReactstrap__NavbarToggler.re#L6

**Comment by liubko** [comment]
just need to change `type_` to `_type`

https://reasonml.github.io/en/try.html?rrjsx=true&reason=NoAQRgzgdA9mBWBdAUAUwB4BdUCcB2AhgDYAEAtgQNaoAKOMADhCQFwkAUyJJAfgPqYAng1RsImHAEs8AcwC8AfgA0XXhEkAvUSXFTZildwCueSZmQBKEnIB8JPtZIAiJwG5kyIqkwkCjitR0jBDs-EIick7Y4k5KHBYWrkA

**Comment by liubko** [comment]
@couchemar do you want to submit a PR with this change?

**Comment by liubko** [comment]
more info here https://bucklescript.github.io/docs/en/object-deprecated.html#name-mangling

**Comment by couchemar** [comment]
@liubko Yes, I'll fix this soon

**Comment by liubko** [comment]
published 0.2.1
